src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.exceptions.InvalidOperationException;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (transaction.getOperation() != FruitTransaction.Operation.RETURN) {
+            throw new InvalidOperationException("Invalid operation. Should be: 'RETURN', but was: "
+                    + transaction.getOperation() + ".");
+        }
+        if (!Storage.containsKey(transaction.getFruit())) {
+            throw new RuntimeException("Item: '" + transaction.getFruit()
+            + "' doesn't exist in the storage");
+        }
+        int currentValue = Storage.getQuantity(transaction.getFruit());

```

Comment:
Direct access to the 'Storage' class is not a good practice. Consider adding a method to the 'Storage' class to update the quantity of items.

Reply:
but I made storage as private, and I also added separate accesss methods...